conconartist/lyric-lava
# Feature/favoriting and testing

#### Purpose: 
> Check all that apply.

### What does this PR do? 
> Clearly and concisely describe changes made. 

Added favoriting prompts feature and new route to display favorite prompts.  Completed testing with happy paths and passed all tests. Added PropTypes.  Refactored functions and cleaned up code.

### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

App.spec.js was updated.  Mockdata.json was updated but not yet implemented into the async testing.  Added Word, FavPrompt, and FavoritePrompts components to complete functionality in .js files.  

### How should this be tested?
> What steps need to be taken to confirm functionality? 

Run `npm start` in terminal when entering the project directory.  Click buttons, add prompts, enter search terms to see full functionality of site.  

Run `npx cypress open` in root of project directory to run cypress tests for App.spec.js in integration folder.


#### Checklist:
